@navikt/core/react/src/form/error-summary/ErrorSummary.tsx

```diff
@@ -105,7 +107,7 @@ export const ErrorSummary = forwardRef<HTMLDivElement, ErrorSummaryProps>(
           ref={headingRef}
           tabIndex={-1}
         >
-          {heading}
+          {heading ? heading : translate("Heading")}
```

Comment:
I think this can be simplified like this:
```suggestion
          {heading ?? translate("Heading")}
```